plugins/tokenizers/mecab.c

```diff
@@ -1089,12 +1096,8 @@ mecab_next_wakati_format(grn_ctx *ctx,
       break;
     }
 
-    if (space_len > 0) {
-      const char *q = r + space_len;
-      while (q < e && (space_len = grn_isspace(q, encoding))) {
-        q += space_len;
-      }
-      tokenizer->next = q;
+    if ((mecab_next_newline_delimiter(r))) {
+      tokenizer->next = r + space_len;
```

Comment:
Do we need to define `mecab_next_newline_delimiter()`?

```suggestion
    if (cl == 1 && r[0] == '\n') {
      tokenizer->next = r + cl;
```

Reply:
fix: [5f3933c](https://github.com/groonga/groonga/pull/2331/commits/5f3933c66598718a8ef52864a8602b005237513f) Thanks! We don't need it because we just use it here in this moment.